src/Microsoft.AspNet.OData.Shared/Builder/PropertyDescriptor.cs

```diff
@@ -0,0 +1,174 @@
+﻿using Microsoft.AspNet.OData.Common;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+
+namespace Microsoft.AspNet.OData.Builder
+{
+    /// <summary>
+    /// Property descriptor
+    /// </summary>
+    public class PropertyDescriptor
+    {
+        private readonly PropertyInfo _propertyInfo;
+        private readonly MethodInfo _methodInfo;
```

Comment:
And Combine them as one field? 

private readonly MemberInfo _MemberInfo;

Then returns the following properties:

public PropertyInfo PropertyInfo
public MethodInfo MethodInfo 

by  the _MemberInfo

Reply:
fixed